src/index.js

```diff
@@ -96,6 +96,10 @@ class Dropzone extends React.Component {
 
     for (let i = 0; i < max; i++) {
       const file = droppedFiles[i];
+      
+      // Let's add inputName to file, so we can differentiate multiple dropzones
+      file.inputName = this.props.name ? this.props.name : '';
```

Comment:
Wouldn't `file.inputName = this.props.name || '';` be better since `name` is always a String?